examples/svm/plot_weighted_samples.py

```diff
@@ -53,14 +53,13 @@ def plot_decision_function(classifier, sample_weight, axis, title):
 sample_weight_last_ten[9] *= 15
 
 # for reference, first fit without sample weights
+clf_no_weights = svm.SVC(gamma=1)
+clf_no_weights.fit(X, y)
 
 # fit the model
```

Comment:
These changes were also made in this [PR (#21597)](https://github.com/scikit-learn/scikit-learn/pull/21597). I don't know if these will conflict later on.